### PR TITLE
Feature/#206/redefine small design changes

### PR DESCRIPTION
**Pull request by Ccolina03**
-Home Page only allows 3 vendors now

Figma Ans:
-Font size of 'Welcome to ISA's mobile app' is 35 px
-Spacing not 16px. 
-Icon moved to 6.5
-Font size for Vendor Cards info changed to 16,14 and 12 for title, address, and description, respectively (Added +2 to each to match better my screen to the figma)
-Add spacing between address and description
-Added shadow in line for tab bar
-Font size in tab bar icon names to 12
-Adding spacing to Vendors in top (Search Bar page)
-Matching search bar horizontal marginal to 24, as well as Vendors
-Search bar size, position, border radius to 15 (figma)
-Notification text in My ICard is 12 px
-Changed color red to match figma color red
-Font Size of userName and ISAFStatus (Inactive) in Icard is 26 px

Images: 

image

image

image

image

image

### CODE REVIEW

**Comment by ArmaanKatyal** [comment]
Can you please remove the changes to `backend/yarn.lock` that might break the backend. As this PR has no relation to the backend. Thanks

**Comment by ArmaanKatyal** [comment]
I have deleted `package-lock.json` that was being added to the main directory for no reason. If required can be pushed back again.

File: mobile/src/views/HomeView.jsx [hunk 1]
```diff
@@ -56,9 +57,10 @@ const HomeView = ({navigation}) => {
 
         <VendorList
           searchPhrase={searchPhrase}
-          data={vendorData}
+          data={vendorData.slice(0,3)}
```
Comment (ArmaanKatyal) [inline mobile/src/views/HomeView.jsx:61]: why do we need to slice the vendor data?

File: mobile/src/components/shared/VendorList.jsx [hunk 2]
```diff
@@ -49,22 +49,28 @@ const VendorList = ({searchPhrase, setClicked, data}) => {
...
-})
+        justifyContent: 'center',
+      },
+      contentContainer: {
+        justifyContent: 'center',
+        alignItems: 'center',
+      },
+    });
 
 export default VendorList;
```
Comment (ArmaanKatyal) [inline mobile/src/components/shared/VendorList.jsx]:
```suggestion
export default VendorList;

```

File: mobile/src/components/shared/SearchBar.jsx [hunk 3]
```diff
@@ -59,32 +59,34 @@ const SearchBar = ({clicked, setClicked, searchPhrase, setSearchPhrase}) => {
...
     container: {
-        margin: 15,
+        margin: 24,
         justifyContent: "flex-start",
         alignItems: "center",
         flexDirection: "row",
-        width: "90%",
+        width: "87%",
+        height: 40,
+
```
Comment (ArmaanKatyal) [inline mobile/src/components/shared/SearchBar.jsx]: nit: blank lines can de deleted. Use Prettier to format the code(preferred)

File: mobile/src/views/HomeView.jsx [hunk 4]
```diff
@@ -10,7 +10,7 @@ import {options, url} from '../data/vendorMockData';
 import {shuffle} from '../utilites/Shuffle';
 import {getData} from '../data/vendorMockData';
 import {SafeAreaView} from 'react-native-safe-area-context';
-
+import {ScrollView} from 'react-native-virtualized-view'
```
Comment (ArmaanKatyal) [inline mobile/src/views/HomeView.jsx:13]: nit: Formatting
```suggestion
import {ScrollView} from 'react-native-virtualized-view'

```

**Comment by ArmaanKatyal** [comment]
One more thing, only try to push the files that you have changed in the PR. Try not to push package-lock or yarn.lock because then it can cause breaking changes impossible to find as well as the lines changes is in 10,000 😂

File: mobile/src/views/HomeView.jsx (hunk 1 above)
Comment (Ccolina03) [reply mobile/src/views/HomeView.jsx:61]: To limit to 3 the amount of Vendor Cards appearing in HomePage

**Comment by Ccolina03** [comment]
> Can you please remove the changes to `backend/yarn.lock` that might break the backend. As this PR has no relation to the backend. Thanks



> One more thing, only try to push the files that you have changed in the PR. Try not to push package-lock or yarn.lock because then it can cause breaking changes impossible to find as well as the lines changes is in 10,000 😂

Yes, my bad I was playing around with the backend and forgot that I did that long ago. Should I pr again with the changes?

**Comment by ArmaanKatyal** [comment]
> 

You don't have to create a new PR again. just remove the changes that are not required. 👍🏽

**Review by ArmaanKatyal — APPROVED** [review]
lgtm

**Comment by yismailuofa** [comment]
@ArmaanKatyal So to be clear we want to keep the `mobile/yarn.lock` but delete the `backend/yarn.lock` ?

**Comment by ArmaanKatyal** [comment]
> @ArmaanKatyal So to be clear we want to keep the `mobile/yarn.lock` but delete the `backend/yarn.lock` ?

Yes, carlos by mistake pushed his experimental changes for backend when he was playing around with it. 
His PR is related to frontend so their shouldn't be any changes related to backend. 
@yismailuofa

**Comment by yismailuofa** [comment]
> Yes, carlos by mistake pushed his experimental changes for backend when he was playing around with it.

So we need to remove backend/yarn.lock from the changes then right?

**Comment by ArmaanKatyal** [comment]
> > Yes, carlos by mistake pushed his experimental changes for backend when he was playing around with it.
> 
> 
> 
> So we need to remove backend/yarn.lock from the changes then right?

Yes 👍

File: backend/yarn.lock
Comment (yismailuofa) [inline backend/yarn.lock:1]: As per @ArmaanKatyal we shouldn't delete this, so let's take it out of this PR. Otherwise lgtm

**Review by yismailuofa — CHANGES_REQUESTED** [review]

**Review by yismailuofa — APPROVED** [review]

**Pull request closed without merging**